src/extension/_locales/fr/messages.json

```diff
@@ -1,501 +1,503 @@
 {
-  "title": {
-    "message": "AEM Sidekick"
+  "add": {
+    "message": "Ajouter"
   },
-  "description": {
-    "message": "L'AEM Sidekick permet aux auteurs de simplement visualiser et publier leur contenu."
+  "admin_version": {
+    "message": "Utiliser une version spécifique de l'API Admin"
   },
-  "description_shortcut": {
-    "message": "Afficher/cacher le Sidekick"
+  "admin_version_placeholder": {
+    "message": "Version d'Admin : xyz"
   },
-  "homepage": {
-    "message": "https://www.hlx.live/tools/sidekick/"
+  "advanced": {
+    "description": "Mode de configuration avancé",
+    "message": "Avancé"
   },
-  "docs": {
-    "message": "https://www.hlx.live/docs/sidekick"
+  "anonymous": {
+    "message": "Anonyme"
   },
-  "plugins_loading": {
-    "message": "Chargement..."
+  "bulk_confirm_empty": {
+    "message": "Aucun fichier sélectionné"
   },
-  "plugins_empty": {
-    "message": "Aucune action disponible pour cette page"
+  "bulk_confirm_preview_multiple": {
+    "message": "Voulez-vous vraiment prévisualiser les fichiers $1 ?"
```

Comment:
This doesn't look right: `$1` is an integer (number of files), not a string (file names). Please check translations for other languages as well.

Reply:
@rofe this is just related to localized files are sorted, matching English is
```
  "bulk_confirm_preview_multiple": {
    "message": "Are you sure you want to preview $1 files?"
  },
```

Reply:
@rgeraghty I meant the translation that is wrong (in multiple languages: 
`$1` is a number, so for an English string:
`Are you sure you want to preview 7 files?`
you'd get the following in French:
`Voulez-vous vraiment prévisualiser les fichiers 7 ?`
while the expected output is:
`Voulez-vous vraiment prévisualiser 7 fichiers ?`

Reply:
Thanks @rofe I see, we are correcting

Reply:
@rofe the translators has gone over these and corrected, example for French and German:
```
Voulez-vous vraiment prévisualiser $1 fichiers ?

Möchten Sie wirklich eine Vorschau von $1 Dateien anzeigen?
```

Reply:
@rofe we have passed your comment to vendor/translators to update

Reply:
@rofe we have update the translations, per your comment for 'generating the preview'. The translators update all locales, not just German.